drivers/sx1280/include/sx1280_params.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2022 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_sx1280
+ *
+ * @{
+ * @file
+ * @brief       Default configuration
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ */
+
+#ifndef SX1280_PARAMS_H
+#define SX1280_PARAMS_H
+
+#include "board.h"
+#include "sx1280.h"
+#include "sx1280_constants.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/**
+ * @name    Set default configuration parameters
+ *
+ * Default values are adapted for mbed shield used with to nucleo64 boards
```

Comment:
Is there any off-the-shelf module? Would be nice to set the defaults according to the most easily available hardware, or at least one that you have tested yourself, is there for example a Nucleo shield?